one_shot_scripts/create_local_files.js

```diff
@@ -1,4 +1,4 @@
-import fs from 'fs';
+const fs = require('fs');
```

Comment:
Trying to run this with import gave an error about only modules being allowed to use import. The other one shot scripts use require and that made it executable, so even though my experience with ES6 imports vs CommonJS is minor this seems the correct resolution